src/3box.js

```diff
@@ -221,6 +284,7 @@ class Box extends BoxApi {
    * @property {String} DID        the DID of the user
    */
   get DID () {
+    if (!this._3id) throw new Error('DID: auth required')
```

Comment:
could probs add this constraint to some func/class modifier/wrap, but probably clearer and simpler to keep as is for now

Reply:
yeah, effort vs gain :)